### DISCUSSION TITLE
Anchor Discord

### DISCUSSION THREAD

**Discussion by Rabeebaqdas** [otter-sec/anchor#2981] [General]
Hello folks, hope you all doing well, I want to join the anchor discord group as I am coming from the Ethereum ecosystem to learn Solana development but I have been unable to join the discord group of anchor, for the past few weeks I have stuck on the verify page, can you guys help me out with this, I got the discord link from their website so I don't think the link is incorrect.

**Comment by acheroncrypto** [comment]
Please see https://github.com/coral-xyz/anchor/issues/2960#issuecomment-2106764021
